plugin/storage/es/factory.go

```diff
@@ -218,7 +218,8 @@ func (f *Factory) CreateArchiveSpanReader() (spanstore.Reader, error) {
 	if !f.archiveConfig.Enabled {
 		return nil, nil
 	}
-	sr, err := createSpanReader(f.getArchiveClient, f.archiveConfig, true, f.logger, f.tracer)
+	// TODO: should use_aliases be always set to true here?
```

Comment:
@yurishkuro thoughts on this? is it okay to just respect the user input here?

Reply:
no, I think you need to override `use_aliases=true` here, because it may not be set via CLI flags. jaeger-v2 won't go through this code path at all, so for v2 user input would be respected.